tests/restarting/from_7.1.0_until_7.2.0/ClientMetricRestart-2.toml

```diff
@@ -0,0 +1,13 @@
+[[test]]
+testTitle='ClientMetricRestartTest'
+clearAfterTest=false
+
+    [[test.workload]]
+    testName='ClientMetric'
+    toSet=false
+    testDuration=500.0
+    trInfoSizeLimit=199123
```

Comment:
Curious, how is this number chosen?

Reply:
Ah it's just some random number I invent and I then look for it in test when reading, to make sure global config has what I provided